members/M000303.yaml

```diff
@@ -99,6 +101,8 @@ contact_form:
     - click_on:
       - value: Submit
         selector: div.buttonHolder input.btn-primary
+    - wait:
+      value: 3
```

Comment:
needs indent + dash

Reply:
These changes are for Mccain's yaml. It's here as a mistake. Please see the other PR

Reply:
same as above - still need to take this edit out of this branch

Reply:
Sounds good. How do I take it out tho? Should I close this PR and create a new one instead?